src/extensions/mvs/tree/mvs/mvs-tree-primitives.ts

```diff
@@ -75,6 +75,39 @@ const TubeParams = {
     tooltip: OptionalField(nullable(str), null, 'Tooltip to show when hovering over the tube. If not specified, uses the parent primitives group `tooltip`.'),
 };
 
+const ArrowParams = {
+    /** Start point of the tube. */
+    start: RequiredField(PrimitivePositionT, 'Start point of the arrow.'),
+    /** End point of the tube. */
+    end: OptionalField(nullable(PrimitivePositionT), null, 'End point of the arrow.'),
+    /** If specified, the endpoint is computed as start + direction. */
+    direction: OptionalField(nullable(Vector3), null, 'If specified, the endpoint is computed as start + direction.'),
+    /** Length of the arrow. If unset, the distance between start and end is used. */
+    length: OptionalField(nullable(float), null, 'Length of the arrow. If unset, the distance between start and end is used.'),
+    /** Draw an arrow at the start of the arrow. */
+    arrow_start: OptionalField(bool, false, 'Draw an arrow at the start of the arrow.'),
+    /** Height of the arrow at the start. */
+    arrow_start_height: OptionalField(float, 0.1, 'Height of the arrow at the start.'),
+    /** Radius of the arrow at the start. */
+    arrow_start_radius: OptionalField(float, 0.1, ''),
+    /** Draw an arrow at the end of the arrow. */
+    arrow_end: OptionalField(bool, false, 'Draw an arrow at the end of the arrow.'),
+    /** Height of the arrow at the end. */
+    arrow_end_height: OptionalField(float, 0.1, 'Height of the arrow at the end.'),
+    /** Radius of the arrow at the end. */
+    arrow_end_radius: OptionalField(float, 0.1, 'Radius of the arrow at the end.'),
+    /** Draw a tube connecting the start and end points. */
+    show_tube: OptionalField(bool, true, 'Draw a tube connecting the start and end points.'),
```

Comment:
If we call it `show_tube`, then we should also rename `arrow_start` -> `show_arrow_start` and `arrow_end` -> `show_arrow_end`, for the sake of consistency.

Reply:
Good points on the naming, will adjust.